Accessibility/Accessibility/Product.swift

```diff
@@ -7,3 +7,14 @@
 //
 
 import Foundation
+import UIKit
+
+struct Product {
+    let image: UIImage
+    let title: String
+    let price: Double
+    let colors: [String] // "blue", "multi", "white", "lime", "orange"
```

Comment:
We're going to also need a corresponding Image with each one of these colors. One option is to make this a dictionary `[String: UIImage]` or you can create a `Color` object that has a `displayName` & `image` properties.

Reply:
We'll eventually want to remove these comments